biostar/forum/templates/new_post.html

```diff
@@ -41,7 +41,7 @@
                         <div class="required field">
                             <label>{{ form.post_type.label }}</label>
```

Comment:
Labels should include a `for` attribute pointing to the `id` of the element that it is referencing. For example

```
<label for="{{ form.post_type.id_for_label }}">{{ form.post_type.label }}</label>
```

This should be done for all the labels in this form. 
For more information see  https://docs.djangoproject.com/en/3.2/ref/forms/api/#django.forms.BoundField.id_for_label.